src/Modules/UngradedModules.jl

```diff
@@ -146,10 +171,14 @@ Here, `F` and `G` are equal iff their base rings, ranks, and names for printing
 function (==)(F::FreeMod, G::FreeMod)
   # two free modules are equal if the rank and the ring are
   # TODO it this enough or e.g. stored morphisms also be considered?
+  if is_graded(F) && is_graded(G) 
+    return F.R == G.R && F.d == G.d && F.S == G.S
```

Comment:
What is with the case where one compares a graded with an ungraded module? Is this supposed to forget the grading on the first?

Reply:
Thanks for spotting it.